lib/ansible/modules/net_tools/nsupdate.py

```diff
@@ -207,8 +207,18 @@ def __init__(self, module):
         else:
             self.algorithm = module.params['key_algorithm']
 
+        if self.module.params['type'].lower() == 'txt':
```

Comment:
This issue also affects SPF records (even though they are deprecated).
Maybe there are other record types aswell that use text values?

Reply:
This is strictly a personal opinion, but I don't really feel inclined to add code to support a deprecated record type.